src/js/views/Project/Dependencies.jsx

```diff
@@ -88,12 +66,38 @@ function Dependencies({ project, urlPath }) {
     )
   }
 
+  function getAutomations(category, setter) {
+    const url = new URL('/ui/available-automations', globalState.baseURL)
+    url.searchParams.append('project_type_id', project.project_type_id)
+    url.searchParams.append('category', category)
+    httpGet(
+      globalState.fetch,
+      url,
+      ({ data }) => {
+        setter(
+          data.map(
+            ({ automation_name, integration_name, automation_slug }) => ({
+              automationName: automation_name,
+              integrationName: integration_name,
+              automationSlug: automation_slug
+            })
+          )
+        )
+      },
+      ({ error }) => {
+        setErrorMessage(error)
+      }
+    )
+  }
+
   function getDependency(id) {
     return rows.find((row) => row.id === id)?.name
   }
 
   useEffect(() => {
     updateDependencies()
+    getAutomations('create-project-dependency', setCreateAutomations)
+    getAutomations('remove-project-dependency', setRemoveAutomations)
```

Comment:
You can fetch multiple dependency types at the same time (eg, `category=create-project-dependency&category=remove-project-dependency`) ... _however_ it doesn't look like I return the automation category in the response 🤦

Reply:
Yep that's why I did this. I'll see if I can get the endpoint to return it.